pkg/dss/cockroach/subscriptions.go

```diff
@@ -258,6 +258,8 @@ func (c *Store) InsertSubscription(ctx context.Context, s models.Subscription) (
 		count, err := c.fetchMaxSubscriptionCountByCellAndOwner(ctx, tx, s.Cells, s.Owner)
 		if err != nil {
 			c.logger.Warn("Error fetching max subscription count", zap.Error(err))
```

Comment:
Looks good, we can remove the logger.warn since the error will bubble up and be logged anyways